shared/gh/api/gh.api.js

```diff
@@ -54,16 +60,47 @@ define(['gh.api.admin', 'gh.api.app', 'gh.api.authentication', 'gh.api.config',
                     }
                 }
 
-                utilAPI.cachePartials(function() {
-                    // Continue startup
-
-                    // The APIs have now fully initialised. All javascript that
-                    // depends on the initialised core APIs can now execute
-                    return callback(gh);
+                // Get the app configuration
+                getConfig(function() {
+                    // Cache the partials
+                    utilAPI.cachePartials(function() {
+                        // The APIs have now fully initialised. All javascript that
+                        // depends on the initialised core APIs can now execute
+                        return callback(gh);
+                    });
                 });
             });
         };
 
+        /**
+         * Get the app configuration, except if the global admin page is loaded up
+         *
+         * @param  {Function}    callback    Standard callback function
+         * @private
+         */
+        var getConfig = function(callback) {
+            // Don't attempt to fetch any configuration when the global admin UI or QUnit
+            // is loaded up as it has its own way of dealing with configuration
+            /* istanbul ignore else */
+            if ($('body').data('isglobaladminui') || $('body').data('isqunit')) {
```

Comment:
The subheader already uses an attribute called 'isadminui' so I wanted to keep it in line with that. If you still think it might be better I can change what I did here, let me know.